pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarSplitManager.java

```diff
@@ -143,6 +148,10 @@ ManagedLedgerFactory getManagedLedgerFactory() throws Exception {
         try {
             numPartitions = (this.pulsarAdmin.topics().getPartitionedTopicMetadata(topicName.toString())).partitions;
         } catch (PulsarAdminException e) {
+            if (e.getStatusCode() == 401) {
+                throw new PrestoException(QUERY_REJECTED, "Unauthorized");
```

Comment:
Can you improve the error message? For example, adding the topic name to the error message.